website/docs/basics/blocks-special-use-cases.md

```diff
@@ -6,5 +6,84 @@ sidebar_label: Special Use Cases
 
 [![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
 
+By special use cases, we mean just that **special**.
+For example, in our clients projects, we saw that sometimes you need to provide something special to your blocks, like changing the block editor block DOM structure or change attributes added inside the inner blocks.
 
-*Coming soon*
+These are some of the examples that we found that we would like to share with you. Here we will list all of them and gradually add more.
+
+## Change innerBlock attributes
+
+**Usage:**
+
+- Change attributes in the block editor and in the frontend.
+- Set attributes in the database.
+
+**Use Case:**
+
+Let's say you have a carousel block, and you have a generic image block that you can use as a standalone block. You create a carousel block and set in its manifest limitation to show only the image block to be inserted as an inner block. This is easy to do; you just add your image block to an `allowedBlocks` array inside the carousel block, and everything works. 
+
+But now you have some default attributes set in that image block that you don't want to be set if the block is added using the innerBlock in the carousel. You can change that using our [overrideInnerBlockAttributes](helpers-javascript#overrideinnerblockattributes) helper. Just add this helper to your block entry point, add the attributes you want to change, and you are all set. 
+
```

Comment:
Maybe add a code example here? I'm unsure what the block entry point refers to. Code example makes this more clear.